logbus/setup/setup.go

```diff
@@ -68,6 +68,11 @@ func (bs *BusSetup) SetDefaultPlatform(platform string) {
 	bs.Formatter.SetDefaultPlatform(platform)
 }
 
+// OrgAndProjectSet returns true if the org and project have already been set.
+func (bs *BusSetup) OrgAndProjectSet() bool {
+	return bs.InitialManifest.GetOrgName() != "" && bs.InitialManifest.GetProjectName() != ""
+}
```

Comment:
The choice of these two specific fields seems very arbitrary to detect whether logstream has started. Should we instead have a `IsLogStreamerStarted` function?

Reply:
That's fair. It's not particularly representative of whether streaming has started. Updated.